src/WebService/Client.php

```diff
@@ -396,13 +396,13 @@ private function handleUnexpectedStatus($statusCode, $service, $path)
     }
 
     /**
```

Comment:
Whoops, looks like this other change from my prior PR snuck into this pull request accidentally.  Hopefully it'll still merge seamlessly to `master`, though.